source/main/commands/dir-project-export.ts

```diff
@@ -56,44 +61,48 @@ export default class DirProjectExport extends ZettlrCommand {
     // Import our RE
     let fnExportRE = getFnExportRE(true) // We want the multiline version.
 
-    // Concat the files
-    let contents: string[] = []
-    for (let file of files) {
-      const fileDescriptor = await this._app.getFileSystem().getFileContents(file)
-      let fileContents: string = fileDescriptor.content
-      // Directly make all image paths absolute to prevent errors if used
-      // in nested project directories (in which this._dir.path is not the
-      // same for all files). Also make the footnotes unique to prevent
-      // assigning errors.
-      fileContents = makeImgPathsAbsolute(file.dir, fileContents)
+    let promiseArr: Array<Promise<MDFileMeta | CodeFileMeta>> = []
+    files.forEach(element => {
+      promiseArr.push(this._app.getFileSystem().getFileContents(element))
+    })
 
-      fileContents = fileContents.replace(fnExportRE, (match, p1: string, offset, string) => `[^${String(file.hash)}${p1}]`)
+    const contentsArr = await Promise.all(promiseArr)
+    const fileContentArr: string[] = []
 
-      contents.push(fileContents)
-    }
-    const finalContents = contents.join('\n\n')
+    contentsArr.forEach((value, index) => {
+      let fileContent = makeImgPathsAbsolute(path.dirname(files[index]), value.content)
+
+      if (!(contentsArr.length === 1)) {
+        fileContent = fileContent.replace(fnExportRE, (match, p1: string, offset, string) => `[^${String(files[index].hash)}${p1}]`)
+      }
 
+      fileContentArr.push(fileContent.trim() + `${EOL}${EOL}`)
```

Comment:
Pandoc is able to just take \n\n instead of, on Windows, using \r\n\r\n, right? If so, I would suggest we just keep it as simple as possible and just use \n. Removes one import and makes this all simpler!